invenio_github/api.py

```diff
@@ -209,11 +209,12 @@ def sync_repo_hook(self, repo_id):
                 pass
 
     def check_sync(self):
-        """Check and sync if over a day has passed since the last sync."""
+        """Check if sync is required based on last sync date."""
         now = utcnow()
-        yesterday = now - timedelta(days=1)
+        expiration_td = current_app.config['GITHUB_SYNC_EXPIRATION_TIMEDELTA']
```

Comment:
Shall we unify the refresh config name with `OAUTHCLIENT_CERN_REFRESH_TIMEDELTA` (https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/contrib/cern.py#L133)

E.g. `GITHUB_REFRESH_TIMEDELTA` or propose new name for `OAUTHCLIENT_CERN_REFRESH_TIMEDELTA`